mlrun/config.py

```diff
@@ -83,7 +83,7 @@
     "images_to_enrich_registry": "^mlrun/*,python:3.9",
     "kfp_url": "",
     "kfp_ttl": "14400",  # KFP ttl in sec, after that completed PODs will be deleted
-    "kfp_image": "mlrun/mlrun",  # image to use for KFP runner (defaults to mlrun/mlrun)
+    "kfp_image": "mlrun/mlrun-kfp",  # image to use for KFP runner (defaults to mlrun/mlrun-kfp)
```

Comment:
one day we will have mlrun-kfp and mlrun-kfp2
do we need to prepare the infrastructure here or could be done once we introduce both images?

Reply:
I think this is out of scope for the current PR,